lib/crystalball/source_diff/file_diff.rb

```diff
@@ -10,7 +10,7 @@ def initialize(git_diff)
       end
 
       def moved?
-        git_diff.patch =~ /rename from.*\nrename to/
+        !(git_diff.patch =~ /rename from.*\nrename to/).nil?
```

Comment:
do we need a spec for it? I guess you faced an issue here

Reply:
Not an issue. It works like a charm now. It is just a minor fix. It is inconvenient to return `Integer` here I think.